src/framework/lib/genesisframework/utils.rb

```diff
@@ -46,9 +47,19 @@ def self.log subsystem, message
         puts logline
         Syslog.open("genesis", Syslog::LOG_PID, Syslog::LOG_USER) unless Syslog.opened?
         Syslog.log(Syslog::LOG_INFO, logline)
```

Comment:
I was writing this with external systems in mind, and syslog will always be available. But yeah, it would probably be a pretty good fit for moving out of `Utils` as well. Should be pretty simple too.